### DISCUSSION TITLE
Office365 Builds

### DISCUSSION THREAD

**Discussion by CarlosAmericano** [AshleyHow/WindowsOSBuild#41] [Ideas]
It would be nice to have a similar module to parse the M365 apps builds 
https://learn.microsoft.com/en-us/officeupdates/update-history-microsoft365-apps-by-date

As part of this or a different module altogether.

**Comment by AshleyHow** [comment]
Thanks for the suggestion - I'll put it on the to-do list, it shouldn't be
too hard.
